test/Chainweb/Test/SPV.hs

```diff
@@ -343,12 +342,12 @@ spvTransactionOutputRoundtripTest rdb v step = do
 
 type TestClientEnv_ cas = TestClientEnv MockTx cas
 
-apiTests :: RocksDb -> Bool -> ChainwebVersion -> TestTree
-apiTests rdb tls v = withTestPayloadResource rdb v 100 (\_ _ -> return ()) $ \dbIO ->
```

Comment:
This change is unrelated. The `tls` parameter was always `True` and thus both test case groups were actually doing the same.